src/components/v5/common/Table/types.ts

```diff
@@ -1,7 +1,8 @@
-import { TableOptions } from '@tanstack/react-table';
+import { TableOptions, Row } from '@tanstack/react-table';
 
 export interface TableProps<T>
   extends Omit<TableOptions<T>, 'getCoreRowModel'> {
   getCoreRowModel?: TableOptions<T>['getCoreRowModel'];
   className?: string;
+  getRowClassName?: (row: Row<T>) => string | undefined;
```

Comment:
getRowClassName?: (row: Row<T>) => string;

Reply:
The type is correct here. `?` doesn't mean that the function can also return undefined